plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/BaseMongoDistributedQueries.java

```diff
@@ -78,11 +76,8 @@ protected TestTable createTableWithDefaultColumns()
     public void testColumnName(String columnName)
     {
         if (columnName.equals("a.dot")) {
-            // TODO (https://github.com/trinodb/trino/issues/3460)
-            assertThatThrownBy(() -> super.testColumnName(columnName))
-                    .hasStackTraceContaining("TableWriterOperator") // during INSERT
-                    .hasMessage("Invalid BSON field name a.dot");
-            throw new SkipException("Insert would fail");
+            new SkipException("Insert would fail");
```

Comment:
Please don't remove the existing assertThatThrownBy. We should test with new error message.

Reply:
You told me to remove it

Reply:
https://github.com/trinodb/trino/pull/9819#discussion_r746462564

Reply:
The comment doesn't mean removing. Please keep both assertThatThrownBy and SkipException as the original code.